DataFormats/Luminosity/interface/PCC.h

```diff
@@ -0,0 +1,56 @@
+#ifndef PCC_PCC_h
+#define PCC_PCC_h
```

Comment:
Please use an include guard matching the full file name subsystem_package_fileName_h